tests/difference/core/model/src/model.ts

```diff
@@ -422,6 +422,13 @@ class CCVProvider {
   };
 
   onReceive = (data: PacketData) => {
+
+    // Drop slash packets for double-sign infraction
+    if ('isDowntime' in data && ! data.isDowntime) {
```

Comment:
Curious why these two checks involve downtime if the code is intended to drop double-sign packets?

Reply:
Yea, this is a bit confusing. PacketData is a union type in TS, meaning that it can either carry slash or VSC mature data. So the first check asserts that the `isDowntime` field exists and the second one asserts that the infraction isn't for downtime but for a double sign. It should definitely be improved by using an enum or adding a `isDoubleSign` field.

Reply:
To me, it looks like `data.isDowntime` will be true if it is IS a downtime packet, and thus the negation `!data.isDowntime` will be true if it IS NOT a downtime packet. So the code inside the conditional will be executed if the packet IS NOT a downtime packet, yet it seems to fire the downtime event

Reply:
I believe that you probably intended to fire a double sign event not a downtime event

Reply:
True good catch

Reply:
LGTM now, resolving comment